Tests/BFKitTests/Linux/Foundation/ArrayExtensionTests.swift

```diff
@@ -64,7 +64,6 @@ class ArrayExtensionTests: XCTestCase {
                 XCTAssert("\(i)" != "\(element)")
```

Comment:
What are you testing here?
This is what your test is doing: `if i == element { return i == element } else { return i != element }`.
You are always return true.
If you want to find the random element inside the array you can remove the `else`, add again the `XCTFail()` and add a `return` after the `XCTAssert`